src/config/cli.rs

```diff
@@ -22,9 +25,9 @@ pub struct Cli {
     pub command: Option<Command>,
 
     // General Options
-    #[arg(short, long, help_heading = "General Options")]
-    /// Path to the configuration file. If not supplied, defaults will be used.
-    pub config: Option<String>,
```

Comment:
Is this a breaking change? If so, maybe we want to smoothen the transition, e.g. just yield a warning when this option is used?

Reply:
I will add a warning for good measure but it was a very recent addition that I dont think is used, and does not align with anvil.